app/src/androidTest/java/org/projectbuendia/client/user/UserManagerTest.java

```diff
@@ -52,19 +54,24 @@ public void testLoadKnownUsers_GeneratesEventOnSucess() throws Exception {
         mFakeAsyncTaskRunner.runUntilEmpty();
         // THEN the user manager fires off a KnownUsersLoadedEvent
         assertTrue(mFakeEventBus.getEventLog().contains(
-            new KnownUsersLoadedEvent(ImmutableSet.of(USER))));
+                new KnownUsersLoadedEvent(ImmutableSet.of(USER))));
     }
 
     /** Tests that an event is posted when users fail to load. */
     public void testLoadKnownUsers_GeneratesEventOnFailure() throws Exception {
-        // GIVEN the user store returns an empty set of users
-        when(mMockUserStore.loadKnownUsers()).thenReturn(ImmutableSet.<JsonUser> of());
+        // GIVEN the user store throws an exception when trying to load the users
+        when(mMockUserStore.loadKnownUsers()).thenAnswer(new Answer<Object>() {
+            @Override
+            public Object answer(InvocationOnMock invocationOnMock) throws Throwable {
+                throw new InterruptedException("INTENDED FOR TEST");
+            }
+        });
         // WHEN loadKnownUsers is called and the async task is run
         mUserManager.loadKnownUsers();
         mFakeAsyncTaskRunner.runUntilEmpty();
         // THEN the user manager fires off a KnownUsersLoadFailedEvent
         mFakeEventBus.assertEventLogContains(
-            new KnownUsersLoadFailedEvent(KnownUsersLoadFailedEvent.REASON_NO_USERS_RETURNED));
+                new KnownUsersLoadFailedEvent(KnownUsersLoadFailedEvent.REASON_UNKNOWN));
     }
```

Comment:
Well done. I've realized yesterday that this method was not passing but i had no time fix it. 
 Probably this is a auto-indentation. It has 4 spaces more than it should has.